buildenv/jenkins/common/build.groovy

```diff
@@ -326,8 +326,63 @@ def archive_sdk() {
     }
 }
 
+/*
+* When crash occurs in java, there will be an output in the log: IEATDUMP success for DSN='JENKINS.JVM.JENKIN*.*.*.X&DS'.
+* The X&DS is a macro that is passed to the IEATDUMP service on z/OS.
+* The dump is split into multiple files by the macro if the address space is large.
+* Each dataset is named in a similar way but there is a distinct suffix to specify the order. 
+* For example, JVM.*.*.*.001, JVM.*.*.*.002, and JVM.*.*.*.003.
+* The function fetchFile moves each dataset into the current directory of the machine, and appends the data to the core.xxx.dmp at the same time.
+* The file core.xxx.dmp is uploaded to artifactory for triage.
+*/
+def fetchFile(src, dest) {
+    // define the name of the dataset to move
+    def fileToMove = src.substring(src.indexOf('.') + 1)
```

Comment:
Shouldn't this use lastIndexOf?

Reply:
Hey Peter. That's a good point. But the src that is passed to this function fetchFile is in this form: Jenkins.JVM.*.*001. However, the actual file name on the system is JVM.*.*001. So we will use indexOf to remove Jenkins. Thanks! @pshipton

Reply:
Although....Where does `Jenkins` come from? The username of the system user? If that is the case then maybe `$USER` might be smarter than hardcoding `Jenkins`

Reply:
Hey @AdamBrousseau! Thanks for your advice. I am not sure why it begins with Jenkins. I took the string after /IEATDUMP success for DSN=/ and it has the template in this format: 
Jenkins.JVM*. :)

Reply:
> Using the user name is correct, although need to ensure the case of the macro matches the case in the filename, or ignore case.
> Please expand on the comment, i.e. "Remove the user name from the dataset name"

Sorry, I think the one for the macro is also uppercase and it matches the variable USER. I have fixed the code and the comment. Thanks for checking! @pshipton